slack-clone-compose-sample/src/main/kotlin/io/getstream/slack/compose/ui/MainActivity.kt

```diff
@@ -0,0 +1,22 @@
+package io.getstream.slack.compose.ui
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+import io.getstream.slack.compose.ui.home.HomeScreen
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            ChatTheme {
+                Surface(color = MaterialTheme.colors.background) {
```

Comment:
We should avoid using the `MaterialTheme` as we have our own wrapper! Try using the `ChatTheme.colors` instead :]